store.html

```diff
@@ -0,0 +1,144 @@
+                <li><a class="nav-li" href="#">Store</a></li>
+                <li><a class="nav-li" href="#">Blog</a></li>
+                <li><a class="nav-li" href="#">Help</a></li>
+                <li><a class="nav-li" href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="form-image-girl-working">
+        <form style="width: 100%;">
+            <legend><h1 class="color-orange">Order your Karma wifi device today!</h1></legend>
+            <fieldset class="container-2-box">
+                <div class="display-flex-column">
+                    <label class="label-padding" for="name">First name *</label>
+                    <input class="      " type="text" id="name" name="name">
+                </div>
+                <div class="display-flex-column">
+                    <label class="label-padding" for="last_name">Last name *</label>
+                    <input class="       " type="text" id="last_name" name="last_name">
+                </div>
+            </fieldset>
+            <fieldset class="display-flex-column">
+                <label class="label-padding" for="address">Address *</label>
+                <input type="text" id="address" name="address">
+            </fieldset>
+            <fieldset class="display-flex-column">
+                <label class="label-padding" for="address">Address 2 *</label>
+                <input type="text" id="address" name="address">
+            </fieldset>
+            <fieldset class="container-city-postcode">
+                <div class="city">
+                    <label class="label-padding" for="city">City *</label>
+                    <select class="select-city" name="city" id="city" required>
+                        <option value="London">Select your city...</option>
+                        <option value="London">London</option>
+                        <option value="Rome">Rome</option>
+                        <option value="Paris">Paris</option>
+                        <option value="Dublin">Dublin</option>
+                        <option value="Edinburgh">Edinburgh</option>
+                    </select>
+                </div>
+                <div class="postcode">
+                    <label class="label-padding" for="postcode">Postcode *</label>
+                    <input class="postcode-input" type="text" id="postcode" name="postcode">
+                </div>
+            </fieldset>
+            <fieldset>
+
+                <label class="label-padding" for="radio">Select a colour</label><br><br>
+                <input class="label-padding" type="radio" value="karmaorange" name="radio" id="karmaorange" required>
+                <label class="label-padding" for="karmaorange">Karma orange</label>
+                <input class="label-padding" type="radio" value="spacegrey" name="radio" id="spacegrey">
+                <label for="spacegrey">Space Grey</label><br><br><br><br>
```

Comment:
Use <br> sparingly. It is very rarely needed. Usually you can achieve much better result with margin/padding.